pkg/proxy/ipvs/proxier_test.go

```diff
@@ -5797,6 +5841,7 @@ func TestDismissLocalhostRuleExist(t *testing.T) {
 }
 
 func TestLoadBalancerIngressRouteTypeProxy(t *testing.T) {
+	t.Parallel()
```

Comment:
just wondering if we should add this for each sub-test rather than test function .. wherever applicable ... 🤔 
```
		t.Run(test.name, func(t *testing.T) {
			t.Parallel()
```

Reply:
No, not unless the `t.Parallel()` on `func Test*` level doesn't affect execution in `t.Run()`. The intention is to run in parallel as much as possible, but without complicate (clutter up) test cases.

Reply:
(most test functions don't call `t.Run()`)

Reply:
ok.